From a13ae1e8b352c33f571b12a1a159f215b3e466b9 Mon Sep 17 00:00:00 2001
From: Michael W <baddaemon87@gmail.com>
Date: Mon, 11 Nov 2024 19:09:56 +0100
Subject: [PATCH 13/98] Settings: Move Google Setting way down

Having it at the very top /might/ make sense on a Google centric device
like a Pixel, but on a neither Google focused device or OS, it should
not be as prominent as it is right now

Change-Id: Iff3dae180e19b32ad684c13b9cf7b0263b7b9e89
---
 res/xml/top_level_settings_expressive.xml | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/res/xml/top_level_settings_expressive.xml b/res/xml/top_level_settings_expressive.xml
index 1748469d07e..73e1a82303c 100644
--- a/res/xml/top_level_settings_expressive.xml
+++ b/res/xml/top_level_settings_expressive.xml
@@ -20,10 +20,6 @@
     xmlns:settings="http://schemas.android.com/apk/res-auto"
     android:key="top_level_settings">
 
-    <com.android.settingslib.widget.UntitledPreferenceCategory
-        android:order="-140"
-        android:key="top_level_account_category"/>
-
     <com.android.settingslib.widget.UntitledPreferenceCategory
         android:order="-130"
         android:key="top_level_connectivity_category">
@@ -235,6 +231,10 @@
             settings:highlightableMenuKey="@string/menu_key_emergency"/>
     </com.android.settingslib.widget.UntitledPreferenceCategory>
 
+    <com.android.settingslib.widget.UntitledPreferenceCategory
+        android:order="-90"
+        android:key="top_level_account_category"/>
+
     <com.android.settingslib.widget.UntitledPreferenceCategory
         android:order="100"
         android:key="top_level_support_category"
